sql/logic_test.go

```diff
@@ -231,7 +237,7 @@ func (t *logicTest) run(path string) {
 	t.srv = setupTestServer(t.T)
 
 	// Make a temporary directory to hold all our on-disk crypto assets.
-	tempDir, err := ioutil.TempDir(os.TempDir(), "TestLogic")
+	tempDir, err := ioutil.TempDir("", "TestLogic")
```

Comment:
because we do that almost everywhere: (from head)
```
git grep 'ioutil.TempDir('
acceptance/cluster/localcluster.go:     l.CertsDir, err = ioutil.TempDir(pwd, ".localcluster.certs.")
acceptance/cluster/localcluster.go:             l.logDir, err = ioutil.TempDir(pwd, ".localcluster.logs.")
acceptance/main_test.go:                logDir, err = ioutil.TempDir(os.TempDir(), "clustertest_")
gossip/simulation/gossip.go:    dirName, err := ioutil.TempDir("", "gossip-simulation-")
server/status_test.go:  dir, err := ioutil.TempDir("", "local_log_test")
sql/logic_test.go:      tempDir, err := ioutil.TempDir(os.TempDir(), "TestLogic")
util/log/main_test.go:  tmpDir, err := ioutil.TempDir("", "logtest")
util/testing.go:        dir, err := ioutil.TempDir("", prefix)
util/testing.go:                dirs[i], err = ioutil.TempDir("", prefix)
```